### DISCUSSION TITLE
MultiCountry App

### DISCUSSION THREAD

**Discussion by jsaristi** [inspireui/support#5574] [New Feature/Ideas Requests]
There should be a way to link a store with a different store in each country.

If in the United States I have a store with the following url https://us.varely.co or https://varely.co/us/ I could choose the United States and see and order only the products of this country and if I am in Spain to be able to choose the products from Spain that would be related to the store with url https://es.varely.co or https://varely.co/es/

And also that at the beginning one can select the country of the store that one wants to enter if the United States or Spain.

**Comment by jsaristi** [comment]
Please pay attention and vote for this very important feature.

**Reply by ecomdeveloperbd** [reply]
It's is very important feature. Good Question

**Reply by jsaristi** [reply]
Yes, I thinks this is important for the app.

**Comment by jsaristi** [comment]
Hello, Any notice from this?

**Comment by Yan-Aleman** [comment]
I use on each vds server, a separate wordpress for each city. They are united only by the domain (each subdomain has its own dns record). The ability to combine mobile applications of different cities in one application is a great idea (and in the application to be able to switch between cities). I am ready to buy a new application for each subdomain, but only if I can combine them into one whole application with a choice of cities.

🙏🙏🙏
